src/config.rs

```diff
@@ -2046,6 +2084,17 @@ impl GooseConfiguration {
             }
         }
 
+        // If set, timeout must be non-zero.
+        if let Some(timeout) = self.timeout.as_ref() {
+            if timeout == "0" {
```

Comment:
Should this not already do a test conversion to check if eg “abc” is passed?

Reply:
It’s validate le already to be an unsigned integer.  Now we validate it’s not 0.

Reply:
actually: I'm wrong -- we were validating in an earlier version but it's currently broken -- I'll fix.

Reply:
fixed so it's properly validating the timeout is > 0.0